app/src/config/index.tsx

```diff
@@ -21,7 +21,8 @@ const chainsMainnet = [mainnet] as const
 
 // Create wagmiConfig
 export const config = defaultWagmiConfig({
-  chains: shouldUseTestnet ? chainsTestnet : chainsMainnet,
+  chains:
+    process.env.NEXT_PUBLIC_ENVIRONMENT === Environment.Testnet ? chainsTestnet : chainsMainnet, // TODO: Figure out how to access envirnoment state here once needed.
```

Comment:
```suggestion
    process.env.NEXT_PUBLIC_ENVIRONMENT === Environment.Testnet ? chainsTestnet : chainsMainnet, // TODO: Figure out how to access environment state here once needed.
```

and what do you mean by "environment state"?

Reply:
That's an obstacle I couldn't find a solution for right now. Maybe @Victor-Salomon has an idea. 
My goal was to store the environment in Zustand state management and use it from there. Here I just load the env variable and it doesnt adapt yet to the state. So no full in-app environment change possible yet.
The reason is that hooks can only be used inside components and not in such a config file.

Reply:
Right now, this config is built on the server side. 

Don't you think we could move it directly to =>` src/context/index.ts` and handle the switch inside this? Since it's a client component, maybe this is doable. 
This needs anyway to be investigated further.

Reply:
Okay let's try that once we need it.